app/helpers/application_helper.rb

```diff
@@ -629,4 +629,17 @@ def cancel_button_component(class_name: nil, id: , value:, data: nil)
       end
     end
   end
+
+  def content_formats(concept_id: nil, acronym: nil, c: nil, selected_format: nil)   
```

Comment:
put it in ontologies_helper.rb file as related to ontologies only  (not for example to users or other models)